nuts/base_tests/napalm_interfaces.py

```diff
@@ -35,21 +35,25 @@ def nuts_extractor(self) -> InterfacesExtractor:
 
 class TestNapalmInterfaces:
     @pytest.mark.nuts("name,is_enabled")
-    def test_is_enabled(self, single_result, name, is_enabled):
+    def test_is_enabled(
+        self, single_result: NutsResult, name: Any, is_enabled: Any
+    ) -> None:
         assert single_result.result[name]["is_enabled"] == is_enabled
 
     @pytest.mark.nuts("name,is_up")
-    def test_is_up(self, single_result, name, is_up):
+    def test_is_up(self, single_result: NutsResult, name: Any, is_up: Any) -> None:
         assert single_result.result[name]["is_up"] == is_up
 
     @pytest.mark.nuts("name,mac_address")
-    def test_mac_address(self, single_result, name, mac_address):
+    def test_mac_address(
+        self, single_result: NutsResult, name: Any, mac_address: Any
+    ) -> None:
```

Comment:
add more specific type

Reply:
Done, read the value types from the demo test bundle ".yaml"-files.